packages/e/element/files/0001-Migrate-data-from-im.riot.Riot-directory.patch

```diff
@@ -0,0 +1,25 @@
+From 3a42f14bafb7b7b3839737c2d275a9e600334a2f Mon Sep 17 00:00:00 2001
+From: Reilly Brogan <reilly@reillybrogan.com>
+Date: Sat, 6 Jan 2024 18:49:26 -0600
+Subject: [PATCH] Migrate data from im.riot.Riot directory
```

Comment:
I don't think this patch actually migrates data from the `im.riot.Riot` directory? It seems to change the fallback to the legacy `~/.config/Riot` directory to `~/.config/Element` (because the new directory is `io.element.Element`, I assume). That would break Element for all users with their config in `~/.config/Riot`.

Seeing the other changes that are needed to do the rename to `io.element.Element`, maybe we should hold off on it until upstream actually renames the app to `io.element.Element`?